radix-transactions/src/validation/transaction_validator.rs

```diff
@@ -500,12 +515,32 @@ impl TransactionValidator {
                 2,
             ));
         }
+
         let transaction_intent = &prepared.signed_intent.transaction_intent;
+        let root_subintent = &prepared.signed_intent.transaction_intent;
```

Comment:
According to our agreed naming scheme, the term `subintent` is reserved for non-root intents.

Either call it `transaction_intent` (which is already used in the line above) or `root_intent`.

Reply:
Will rename